### PR TITLE
[13.0][MIG] maintenance_equipment_sequence

### PR DESCRIPTION
**Pull request by NuriaXifre**
Migration of the module maintenance_equipment_sequence to v13.

### CODE REVIEW

File: maintenance_equipment_sequence/__manifest__.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 {
     "name": "Maintenance Equipment Sequence",
     "summary": """
         Adds sequence to maintenance equipment defined in the equipment's
         category""",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "license": "AGPL-3",
     "author": "Eficent Business and IT Consulting Services S.L.,"
```
Comment (LoisRForgeFlow) [inline maintenance_equipment_sequence/__manifest__.py]: Change to "ForgeFlow"

File: maintenance_equipment_sequence/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -1 +1 @@
-* Adria Gil Sorribes <adria.gil@eficent.com>
+* Adria Gil Sorribes <adria.gil@forgeflow.com>
```
Comment (LoisRForgeFlow) [inline maintenance_equipment_sequence/readme/CONTRIBUTORS.rst:1]: Add yourself

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

Thanks!

**Review by etobella — APPROVED** [review]
LGTM :+1: No major changes

**Review by jarroyomorales — APPROVED** [review]

**Pull request merged**